ui/admin/app/components/form/storage-bucket/minio/index.hbs

```diff
@@ -0,0 +1,341 @@
+      <F.Options>
+        <option disabled hidden selected value=''>
+          {{t 'titles.choose-an-option'}}
+        </option>
+        {{#each @scopes as |scope|}}
+          <option value={{scope.model.id}}>
+            {{scope.model.displayName}}
+          </option>
+        {{/each}}
+      </F.Options>
+    </Hds::Form::Select::Field>
+  {{else}}
+    <InfoField
+      @value={{@model.scopeModel.displayName}}
+      @icon={{if @model.scopeModel.isGlobal 'globe' 'org'}}
+      disabled={{form.disabled}}
+      as |F|
+    >
+      <F.Label>{{t 'resources.storage-bucket.form.scope.label'}}</F.Label>
+      <F.HelperText>
+        {{#if @model.scopeModel.isGlobal}}
+          {{t 'resources.storage-bucket.form.scope.help_global'}}
+        {{else}}
+          {{t 'resources.storage-bucket.form.scope.help_org'}}
+        {{/if}}
+      </F.HelperText>
+    </InfoField>
+  {{/if}}
+
+  {{! Provider / plugin.name }}
+  <Hds::Form::RadioCard::Group @name='plugin_type' @alignment='center' as |G|>
+    <G.Legend>{{t 'titles.provider'}}</G.Legend>
+    <G.HelperText>{{t 'descriptions.choose-a-provider'}}</G.HelperText>
+
+    {{#each @pluginTypes as |pluginType|}}
+      <G.RadioCard
+        @value={{pluginType}}
+        @checked={{eq pluginType @model.compositeType}}
+        @disabled={{(not @model.isNew)}}
+        {{on 'input' (fn @changePluginType pluginType)}}
+        as |R|
+      >
+        <R.Label>{{t
+            (concat 'resources.storage-bucket.plugin-types.' pluginType)
+          }}</R.Label>
+        {{#if (eq pluginType 'aws')}}
+          <R.Icon @name='aws-color' />
+        {{else}}
+          <R.Icon @name='cloud-upload' />
+        {{/if}}
+      </G.RadioCard>
+    {{/each}}
+  </Hds::Form::RadioCard::Group>
+
+  {{! endpoint_url }}
+  <Hds::Form::TextInput::Field
+    name='endpoint_url'
+    @isRequired={{true}}
+    @value={{@model.endpoint_url}}
+    @isInvalid={{@model.errors.endpoint_url}}
+    disabled={{form.disabled}}
+    {{on 'input' (set-from-event @model 'endpoint_url')}}
+    as |F|
+  >
+    <F.Label>{{t 'resources.storage-bucket.form.endpoint_url.label'}}</F.Label>
+    <F.HelperText>{{t
+        'resources.storage-bucket.form.endpoint_url.help'
+      }}</F.HelperText>
+    {{#if @model.errors.endpoint_url}}
+      <F.Error as |E|>
+        {{#each @model.errors.endpoint_url as |error|}}
+          <E.Message>{{error.message}}</E.Message>
+        {{/each}}
+      </F.Error>
+    {{/if}}
+  </Hds::Form::TextInput::Field>
+
+  {{! bucket_name }}
+  <Hds::Form::TextInput::Field
+    name='bucket_name'
+    @isRequired={{true}}
+    @value={{@model.bucket_name}}
+    @isInvalid={{@model.errors.bucket_name}}
+    disabled={{form.disabled}}
+    readOnly={{not @model.isNew}}
+    {{on 'input' (set-from-event @model 'bucket_name')}}
+    as |F|
+  >
+    <F.Label>{{t 'resources.storage-bucket.form.bucket_name.label'}}</F.Label>
+    <F.HelperText>{{t
+        'resources.storage-bucket.form.bucket_name.help'
```

Comment:
Should these helper texts say something else since they reference AWS in the helper text?

Reply:
@ZedLi 
Good catch! I think you are totally right, alsog the field region, access key id and secret access key mention AWS within the helper, I will check with design about these.

Reply:
@ZedLi 
All the helper texts that have any reference to AWS have been updated.